code/modules/spells/spell_code.dm

```diff
@@ -247,6 +248,8 @@ var/list/spells = typesof(/spell) //needed for the badmin verb for now
 			spawn(overlay_lifespan)
 				qdel(spell)
 				spell = null
+	if(spell_flags & INCLUDEUSER)
```

Comment:
This is going to cause issues. Not only does this flag overlap with IGNOREDENSE for the reason that it's only meant to be used in /targeted spells, it also adds the user to lists which should only include turfs for /aoe_turf spells.